assets/js/googlesitekit/widgets/components/WidgetAreaRenderer.test.js

```diff
@@ -275,7 +275,7 @@ describe( 'WidgetAreaRenderer', () => {
 		} );
 	} );
 
-	it( 'should not render widget area with no active widgets', async () => {
+	it( 'should render empty widget area when has an inactive widget', async () => {
```

Comment:
This is not entirely accurate. It should only render a hidden widget area when it has no active widgets. If it has any active widgets, it should not be hidden. It will technically be empty, you're right, but that isn't really something this test needs to be concerned with.
```suggestion
	it( 'should render a hidden widget area when it has no active widgets', async () => {
```

Reply:
Change done @aaemnnosttv  :+1: